Lib9c/Model/Character/CharacterBase.cs

```diff
@@ -103,7 +103,9 @@ protected CharacterBase(CharacterBase value)
             Skills = value.Skills;
             // 버프는 컨테이너도 옮기고,
             Buffs = new Dictionary<int, Buff.Buff>();
+#pragma warning disable LAA1002
             foreach (var pair in value.Buffs)
```

Comment:
How about turning these lines into this?  This must pass the LAA1002 rule without suppressing it.

```csharp
Buffs = value.Buffs.ToDictionary(kv => kv.Key, kv => (Buff.Buff) kv.Value.Clone());
```